src/modules/mavlink/streams/ACTUATOR_OUTPUT_STATUS.hpp

```diff
@@ -49,37 +49,75 @@ class MavlinkStreamActuatorOutputStatus : public MavlinkStream
 
 	unsigned get_size() override
 	{
-		return _act_output_sub.advertised() ? (MAVLINK_MSG_ID_ACTUATOR_OUTPUT_STATUS_LEN + MAVLINK_NUM_NON_PAYLOAD_BYTES) : 0;
+		return _act_output_sub[0].advertised() ? (MAVLINK_MSG_ID_ACTUATOR_OUTPUT_STATUS_LEN + MAVLINK_NUM_NON_PAYLOAD_BYTES) :
+		       0;
 	}
 
 private:
 	explicit MavlinkStreamActuatorOutputStatus(Mavlink *mavlink) : MavlinkStream(mavlink) {}
 
-	uORB::Subscription _act_output_sub{ORB_ID(actuator_outputs)};
+	uORB::Subscription _act_output_sub[2] {{ORB_ID(actuator_outputs), 0}, {ORB_ID(actuator_outputs), 1}};
 
 	bool send() override
 	{
+		static constexpr unsigned max_instances = sizeof(_act_output_sub) / sizeof(
+					_act_output_sub[0]); /* Number of actuator_outputs instances */
+
+		/* Check if any subscriptions have been updated */
+
+		bool updated = false;
+
+		for (unsigned inst = 0; inst < max_instances; inst++) {
+			if (_act_output_sub[inst].updated()) {
+				updated = true;
```

Comment:
Just minor optimization idea, we could break here from the for loop as one updated found, no need to check further as the situation does not change any more.

Reply:
But no need to change, it works fine like that.